python/discord-bots/nextcord-sql/index.py

```diff
@@ -0,0 +1,58 @@
+import nextcord
+from nextcord.ext import commands
+from asyncmy import connect
+
+import os
+from dotenv import load_dotenv
+
+
+TESTING_GUILD_ID = 1234567890 # change to example 
+
+description = """An example bot sparkedhost preinstalls
+
+Author:
+    Github [Jpuf0]
+    Discord Jpuf#0001
+"""
+
+bot = commands.Bot(description=description)
+
+@bot.event
+async def on_ready():
+    global con
+    con = await connect(
+        host="mariadb.sparkedhost.com",
+        port=20012,
+        user="nextcord",
+        password="nextcordpass",
+        database="nextcord",
```

Comment:
Move this to .env as well

Reply:
Also rename the folder from `nextcord-sql` to `nextcord-mysql`, as this implementation is specific to MySQL/MariaDB and would probably require further changes for it to work with other SQL database software (e.g. PostgreSQL), for which we don't provide database servers.